Pyrado/pyrado/logger/experiment.py

```diff
@@ -71,8 +73,25 @@ def __init__(
         :param exp_id: combined timestamp and extra_info, usually the final folder name
         :param timestamp: experiment creation timestamp
         :param base_dir: base storage directory
+        :param include_slurm_id: If a SLURM ID is present in the environment variables, include them in the experiment ID.
```

Comment:
IMO, the advantage of tracking the slurm ID is not as important as the loss in generatlity. Thus, I would like to ask you to remove the include_slurm_id and everything associated to it